### ISSUE TITLE
Please remove the border CSS, it make my eyes uncomfortable 

### ISSUE THREAD

**Issue by sundongmin** [fisheva/Eva-Theme#75]
When I use Rust and show the implementations, the result see the picture below
<img width="562" alt="image" src="https://user-images.githubusercontent.com/17910228/166296044-71c53249-a008-4590-98ba-10bc7685dfa7.png">

the border make my eyes very uncomfortable 
```css
.monaco-tl-contents .monaco-highlighted-label .highlight {
    color: unset !important;
    border: 1px dotted #ffffff;
    box-sizing: border-box;
}
```
I dont't know if this is my own problem, If you(s) also feel uncomfortable, Please remove the border CSS.
I love this theme so much, Please change it, I will be very grateful~

Thinks.

**Comment by fisheva** [comment]
Fixed in 1.8.3.

**Comment by sundongmin** [comment]
Thanks!
